docs/TRANSCRIPTION.md

```diff
@@ -0,0 +1,222 @@
+
+Then, we make a request with the Agora endpoint `https://api.agora.io/v1/projects/${APP_ID}/rtsc/speech-to-text/tasks?builderToken=${builderToken}` and pass in the transcription configuration into the request body. 
+
+You can find a complete list of request body properties within the [Agora Postman collection](https://documenter.getpostman.com/view/6319646/SVSLr9AM#69bd200a-7543-4104-8ccc-415741abbeb7). 
+
+```ts
+const botUid = "2"
+const outputUid = "3"
+const credential = generateCredential()
+const builderToken = await generateRealTimeTranscriptionResource(channel, credential, APP_ID)
+const botToken = await handleGenerateToken({ channel: channel, role: 1, uid: botUid, expireTime: 3600 })
+const outputToken = await handleGenerateToken({ channel: channel, role: 2, uid: outputUid, expireTime: 3600 })
+const url = `https://api.agora.io/v1/projects/${APP_ID}/rtsc/speech-to-text/tasks?builderToken=${builderToken}`
+const body = {
+    "audio": {
+        "subscribeSource": "AGORARTC",
+        "agoraRtcConfig": {
+            "channelName": channel,
+            "uid": botUid,
+            "token": botToken,
+            "channelType": "LIVE_TYPE",
+            "subscribeConfig": {
+                "subscribeMode": "CHANNEL_MODE"
+            }
+        }
+    },
+    "config": {
+        "features": [
+            "RECOGNIZE"
+        ],
+        "recognizeConfig": {
+            "language": "en-US,es-ES",
+            "model": "Model",
+            "output": {
+                "destinations": [
+                    "AgoraRTCDataStream",
+                ],
+                "agoraRTCDataStream": {
+                    "channelName": channel,
+                    "uid": outputUid,
+                    "token": outputToken
+                }
+            }
+        }
+    }
+}
+const res = await makeRequest("POST", url, credential, JSON.stringify(body))
+const data = await res.json()
+const taskId = data.taskId
+
+return sendSuccessfulResponse({
+    taskId: taskId,
+    builderToken: builderToken,
+})
+```
+
+Upon a successful response, you will receive a `taskId`. Both the `taskId` and the `builderToken` will need to be used to stop the transcription, so they must be returned to the caller.
+
+To test this, run your backend using `npm run dev`. You can use cURL to send a `POST` request in your terminal with a body containing `channel`.
+
+```
+curl -X POST http://localhost:4321/api/transcription/start.json \
+  -H "Content-Type: application/json" \
+  -d '{
+    "channel": "test"
+  }'
+```
+
+If your `POST` request is missing information, you will receive a Bad Request response telling you what you missed. If you have a proper request, you will receive a successful response that looks like this:
+
+```
+{"taskId":"<long string>","builderToken":"<another long string>"}
+```
+
+## Stop Transcribing
```

Comment:
How does the developer know if the stop was successfully executed?

Reply:
Added a similar section as the start comand

```
If your `POST` request is missing information, you will receive a Bad Request response telling you what you missed. If you have a proper request, you will receive a successful response with an empty body.
```